README.md

```diff
@@ -15,18 +15,18 @@ Auto Session takes advantage of Neovim's existing session management capabilitie
 :warning: Please note that if there are errors in your config, restoring the session might fail, if that happens, auto session will then disable auto saving for the current session.
 Manually saving a session can still be done by calling `:SaveSession`.
 
-AutoSession now tracks `cwd` changes!  
```

Comment:
This change ws unintentional, but there's really no need for extra white space, so I just left it in